library/src/main/java/io/appwrite/enums/OrderType.kt

```diff
@@ -0,0 +1,6 @@
+package io.appwrite.enums
```

Comment:
We don't really have support for ENUMs as part of the spec file, we should probably work on building a better pattern for it, as some params might be able to leverage from that as well. I also believe the code examples don't support this.

We have the same issue in the Flutter/Dart SDKs.

Reply:
Done :+1: